### DISCUSSION TITLE
How can I enable micrometer for metrics?

### DISCUSSION THREAD

**Discussion by kevinlilendi** [Netflix/dgs-framework#255] [Q&A]
Hi all,

I am new to this framework, and I want to monitor the metrics, so I add this line to my build.gradle
`implementation 'com.netflix.graphql.dgs:graphql-dgs-spring-boot-micrometer:3.10.2'`

what other things I need to do to enable `gql.query` `gql.error` `gql.dataLoader` to show in my /actuator/metrics or other places I can find it? Is there an working example on this one?

Eventually, I want to export it to Datadog, anything I can do about it?

Thanks

**Comment by peterbierman** [comment]
For me adding the dependency was enough.
I did need to call the grapql at least once before the metric was available at : http://localhost:8080/actuator/metrics/gql.resolver

And ofcourse the metric endpoint should be exposed as detailed here:
https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-features.html#production-ready-endpoints-exposing-endpoints
For testing you can use
`management.endpoints.web.exposure.include=*`
